ale_linters/swift/swiftlint.vim

```diff
@@ -1,9 +1,9 @@
 " Author: David Mohundro <david@mohundro.com>
 " Description: swiftlint for swift files
 
-call ale#linter#Define('swiftlint', {
+call ale#linter#Define('swift', {
 \   'name': 'swiftlint',
 \   'executable': 'swiftlint',
-\   'command': g:ale#util#stdin_wrapper . ' .swift swiftlint',
+\   'command': g:ale#util#stdin_wrapper . ' .swift swiftlint lint --path',
```

Comment:
I wonder, swiftlint apparently supports reading from stdin. Can we remove the use of `stdin_wrapper` here and use `--use-stdin` instead? That will likely work better.